paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentSheetLoading.kt

```diff
@@ -0,0 +1,32 @@
+package com.stripe.android.paymentsheet.ui
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.dimensionResource
+import com.stripe.android.paymentsheet.R
+
+@Composable
+internal fun PaymentSheetLoading(
```

Comment:
✍️  (This is my new 🎉 replacement for compose, let me know if you can think of a better emoji to celebrate with)